### ISSUE TITLE
Client not generated

### ISSUE THREAD

**Issue by mg98** [prisma/prisma-client-go#78]
This is the first time I tried to use Prisma. I followed the readme to set up the go project.

`go run github.com/prisma/prisma-client-go generate` does not generate anything for me. I do not get any output off the command. So no errors.

No files were generated in the project. I even ran `sudo find / -name "db_gen.go"` to see if the file was generated anywhere on my system, but it was not.

I am running Go version 1.13.7 on macOS Catalina.

Am I doing something wrong?

By the way: While executing the lift or generate command, the macOS prompt "Do you allow network connections from prisma-cli-darwin" pops up just for a fraction of the second, before I can even confirm. Maybe this gives a hint to something? Idk.

**Comment by steebchen** [comment]
Thanks for the comment. As you might have seen there was not much activity in the Go client in the last few months, because I was needed on other other parts of Prisma. I'm slowly getting back to Go development though.

Unfortunately, right now you have to specify an absolute path in the `provider` part to the Go executable, e.g. `/Users/luca/go/prisma-client-go`. I'm working on a fix to improve this and looking forward to publish that fix next week. 

About the "Do you allow network connections...?": also a bug which will probably fix itself when I'll update internal Prisma tooling to the latest version.

**Comment by mg98** [comment]
That's alright. Thanks for the quick reply. Can you explain how/ where I can find that setting 😅 I couldn't find anything related through my IDE and Google didn't help me neither.

**Comment by steebchen** [comment]
Not sure what setting you mean. If you mean the absolute path, you can type `which prisma-client-go` to find out the absolute path which you need to specify in the provider.

**Comment by robojones** [comment]
First you need to make sure that the client is installed:
```
go get github.com/prisma/prisma-client-go
```

In your Prisma schema you need to change the generator section to something like this:
```prisma
generator db {
  provider = "/Users/luca/go/bin/prisma-client-go"
  output = "./db/db_gen.go"
  package = "db"
}
```
You need to replace "luca" with your own user name.

**Comment by mg98** [comment]
Oh sorry I didn't ready carefully. So yeah thanks it works now!

**Comment by steebchen** [comment]
Hey @mg98, this should be now fixed via #81! Please follow the instructions in the [v0.0.1 release](https://github.com/prisma/prisma-client-go/releases/tag/v0.0.1) to make upgrade your project accordingly.
